### ISSUE TITLE
Add ARDRegressor

### ISSUE THREAD

**Issue by Thilakraj1998** [blobcity/autoai#65]
Add ARDRegressor model into the library.

Primary File to Change: https://github.com/blobcity/autoai/blob/main/blobcity/config/regressor_config.py

Reference ARDRegressor Implementation:
https://github.com/blobcity/ai-seed/blob/main/Regression/Linear%20Models/ARDRegressor.ipynb

Official API Reference for Parameter:
https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.ARDRegressor.html

Dependencies if any, must be appropriately added. Test run of the train function on a regression problem must pass, and the function must attempt to train an ARDRegressor as a potential best fit model.

**Comment by SaharshLaud** [comment]
I am willing to work on this issue.
Can you please assign it to me @Thilakraj1998 Sir?
